package.json

```diff
@@ -97,6 +97,7 @@
     "config": "3.3.0",
     "connect-session-sequelize": "6.0.0",
     "connect-slashes": "^1.4.0",
+    "cross-env": "7.0.2",
```

Comment:
Mind installing this under `devDependencies`? You can do this by running `npm install --save-dev cross-env` (you may need to uninstall the current version first). This makes sure we don't ship the dependency to production if we don't need it there.

Reply:
(just ran `npm install --save-dev cross-env` locally and confirms that it cleans things up on its own, no no other step needed)